### ISSUE TITLE
Search bar does not return any results

### ISSUE THREAD

**Issue by MasssiveJuice08** [oleeskild/digitalgarden#264]
My digital garden website: [Morrowind Modding with MassiveJuice](https://morrowind-modding-with-massivejuice.vercel.app/)

The search bar does not return any results on [Release 1.61.0](https://github.com/oleeskild/digitalgarden/releases/tag/1.61.0). Tested in latest versions of Google Chrome and Microsoft Edge. No results are given whether searching for page title or tag.

![Screenshot 2024-05-21 120308](https://github.com/oleeskild/digitalgarden/assets/122503558/2b50d00b-e3cf-4f17-8ebe-f101b675cf55)

**Comment by oleeskild** [comment]
I've addressed this in the discord, but reposting here for reference:

>Looked into this, it's because of the note "N'Gasta Pasta - Home". It seems the code doesn't handle titles with the symbol ' in them 😅  I'll see if what I can do to fix it. In the meantime, would it be possible for you to try to delete that note, or remove the " ' ", and see if that fixes it?


>Okay, that was just part of the problem. You found another edgecase: using html tags in the title, e.g. "<acronym>" and "<abbr>", also crashes it. I've yet to figure out why it can't handle it, I will need to look further into it. But try removing those as well and see if that helps.

TL;DR: Single apostrophes and html tags in note titles breaks the search.

**Comment by MasssiveJuice08** [comment]
Fixed for me now but removing the apostrophes and HTML tags from note titles. You may optionally want to leave this issue open for tracking a fix for search being able to parse those
